nautobot_ssot/integrations/dna_center/utils/dna_center.py

```diff
@@ -198,25 +198,6 @@ def get_port_status(port_info: dict):
             status = "Planned"
         return status
 
-    @staticmethod
-    def parse_hostname_for_role(hostname_map: List[Tuple[str, str]], device_hostname: str):
```

Comment:
Can we get a quick change log entry for this change?

Reply:
Done!